README.md

```diff
@@ -1,2 +1,58 @@
-# to-do-list
+# To-Do List
 To-Do List is an app that list tasks to be done.
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- Webpack
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps inside the root folder.
+
+### Setup
+
+Install dependencies with:
+
+```
+npm run build
```

Comment:
- [ ] Kindly replace this with `npm install` in order to install all dependencies.